scripts/generate-patch

```diff
@@ -88,7 +65,13 @@ for f in packages/*; do
 			if [[ -d ${f}/charts ]]; then
 				split_crds=$(cat ${f}/package.yaml | yq r - generateCRDChart.enabled)
 				if [[ "${split_crds}" == "true" ]]; then
-					revert_crd_changes ${f}
+					./scripts/clean-crds ${f}
+				fi
+				providesGVR="$(cat ${f}/package.yaml | yq r - providesGVR)"
+				if ! [[ -z ${providesGVR} ]]; then
+					if [[ "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/provides-gvr]')" == "${providesGVR}" ]]; then
+						yq d -i ${f}/charts/Chart.yaml "annotations[catalog.cattle.io/provides-gvr]"
+					fi
```

Comment:
Another change from needed to fix https://github.com/rancher/charts/pull/604 to ensure that the flag is removed on generating a patch.